### ISSUE TITLE
Hey! Awesome work

### ISSUE THREAD

**Issue by itztopcat** [platoathens/Reforger-Shop-System#3]
I am currently building a sector control scenario(PvP)

My question is; Is it possible to add a logic that creates currency if a player in on the objective? 

Lets say: If a player is on the triggering objective some cash is added to the player or, when a player kills another player, some cash is added to the player.

**Comment by ekudmada** [comment]
Yes, you can do that yourself in your gamemode. The shop system itself won't have the ability to do that for you. If you check ADM_CurrencyComponent you will see all of the logic for adding/removing currency from the player. Those functions only work if an item which acts as currency already exists, so you'll want to add a check to see if they already have a wallet or some custom currency item you've created and if they don't add it to their inventory before giving them money.
